pycheribuild/projects/cross/opensbi.py

```diff
@@ -82,6 +83,8 @@ def setup(self):
         super().setup()
         compflags = " " + self.commandline_to_str(self.essential_compiler_and_linker_flags)
         compflags += " -Qunused-arguments"  # -mstrict-align -no-pie
+        if self.config.riscv_cheri_isa == RiscvCheriISA.STD:
+            compflags += " -mno-relax"
```

Comment:
Shouldn't this be always-on now?

Reply:
(mandated by the compiler driver that is)

Reply:
Hmm we have this line:

```
            result.append(
                "-mrelax" if _linker_supports_riscv_relaxations(instance.linker, config, xtarget) else "-mno-relax"
            )
```
Should probably just delete that and let the compiler driver decide.

Reply:
ah and that function has
```
    if xtarget.is_hybrid_or_purecap_cheri():
        return False
```
so yeah this can be dropped.